src/gluonts/nursery/daf/tslib/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The code is based on basic code developed on my own based on pytorch and is not compatible with gluonts. Consolidating requires an overhaul by replacing the current structure with gluonts style training. It may take a while.